crates/primitives/src/withdrawal.rs

```diff
@@ -1,8 +1,7 @@
-use std::mem;
-
-use crate::{constants::GWEI_TO_WEI, serde_helper::u64_hex, Address, U256};
+use crate::{constants::GWEI_TO_WEI, serde_helper::u64_hex, Address};
 use reth_codecs::{main_codec, Compact};
 use reth_rlp::{RlpDecodable, RlpEncodable};
+use std::mem;
```

Comment:
needed?

Reply:
Change is before this PR, it was just a refactor.